modules/consentManagement.js

```diff
@@ -220,7 +219,7 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
     window.addEventListener('message', readPostMessageResponse, false);
 
     // call CMP
-    window[apiName](commandName, null, moduleCallback);
+    window[apiName](commandName, moduleCallback);
```

Comment:
@Josh-G do I understand correctly that changing `null` to `undefined` fixes this for Quantcast? 
```
window[apiName](commandName, undefined, moduleCallback);
```

Reply:
if so, sounds like this is the change we should make to preserve the function signature.

Reply:
llooks like an official quantcast response below that @harpere is correct